mkdocs.yml

```diff
@@ -120,6 +120,7 @@ nav:
         - Projects: tutorials/projects.md
         - Deployments: tutorials/deployments.md
         - Storage & Infrastructure: tutorials/storage.md
+        - Developing a New Worker Type: tutorials/developing-a-new-worker-type.md
```

Comment:
I think this got litigated already but I could see this under `Guides` as opposed to tutorials